lib/matplotlib/pyplot.py

```diff
@@ -2155,17 +2154,15 @@ def matshow(A, fignum=None, **kwargs):
 
     """
     A = np.asanyarray(A)
-    if fignum is False or fignum is 0:
+    if fignum == 0:
```

Comment:
This is a behavior change for the case of `fignum==False`.

Reply:
No:
```
In [1]: False == 0                                                                                                                                                                                  
Out[1]: True
```

Reply:
fair enough.